scripts/app.js

```diff
@@ -0,0 +1,119 @@
+const bookArray = []; // global array
+
+// MVP part
+/**
+ * This is the booj object
+ * @param {string} title The title of the book
```

Comment:
What is the purpose of lines 6-9 (usage of `@param`)?

Reply:
Azdren was showing me how eslint rules work,

Reply:
Lines from 6 to 9 explain the parameters of the book object this js-doc comment is enforced by eslint we can add a rule into eslint config JSON file to ignore it